src/utils/validators_enhanced.py

```
@@ -0,0 +1,479 @@
+# Backward compatibility functions (return bool like original)
+def validate_email(email: str) -> bool:
+    """Backward compatible email validation function with provider check."""
+    try:
+        # Сначала базовая валидация
+        result = validate_email_enhanced(email)
+        if not result:
+            return False
+        
+        # Проверка провайдера
+        from .email_providers import validate_email_provider
+        is_valid_provider, error_msg = validate_email_provider(email)
```

Ruff (0.14.10)

Unpacked variable `error_msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)